AzureMonitorAgent/agent.py

```diff
@@ -369,7 +372,13 @@ def enable():
     """
     exit_if_vm_not_supported('Enable')
 
-    OneAgentEnableCommand = "systemctl start mdsd"
+    public_settings, protected_settings = get_settings()
+
+    if public_settings.get("AUTO_CONFIG") == "true":
+        OneAgentEnableCommand = "systemctl start mdsdmgr"
+    else:
+        OneAgentEnableCommand = "systemctl start mdsd"
```

Comment:
For the default tenant (/etc/default/mdsd), you need mdsd service running.

Has Nikhil confirmed to move /etc/default/mdsd to /etc/mdsd.d/tenants?